vault/resource_pki_secret_backend_sign_test.go

```diff
@@ -31,18 +31,28 @@ func TestPkiSecretBackendSign_basic(t *testing.T) {
 		CheckDestroy: testCheckMountDestroyed("vault_mount", consts.MountTypePKI, consts.FieldPath),
 		Steps: []resource.TestStep{
 			{
-				Config: testPkiSecretBackendSignConfig_basic(rootPath, intermediatePath),
+				Config: testPkiSecretBackendSignConfig_basic(rootPath, intermediatePath, ""),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(resourceName, "backend", intermediatePath),
 					resource.TestCheckResourceAttr(resourceName, "common_name", "cert.test.my.domain"),
 					testValidateCSR(resourceName),
 				),
 			},
+			{
+				SkipFunc: func() (bool, error) {
```

Comment:
These tests will be pulled into their own tests in a follow-up PR since Root Issuers need to be imported into the Intermediate mount for the full workflow. It will be better to isolate the test steps into their own test when the import bit is added in. Going to leave these in for this PR to demonstrate that the field is working as expected